.github/workflows/docker-publish.yaml

```diff
@@ -90,3 +90,21 @@ jobs:
             ${{ steps.login-ecr.outputs.registry }}/lakefs:${{ steps.version.outputs.tag }}
             treeverse/lakefs:${{ steps.version.outputs.tag }}
             treeverse/lakefs:latest
+
+      - name: Trigger playground deploy
+        run: |
+          curl \
+          -X POST \
+          -H "Authorization: Bearer ${{ secrets.PAT_TOKEN }}"\
+          -H "Accept: application/vnd.github+json" \
+          -H "Content-Type: application/json" \
+          https://api.github.com/repos/${{ secrets.REPOSITORY_NAME }}/actions/workflows/${{ secrets.WORKFLOW_ID_STAGING }}/dispatches \
+          --data '{"ref":"main", "inputs": { "lakefs_image": ${{ steps.version.outputs.tag }} }}'
+
+          curl \
+          -X POST \
+          -H "Authorization: Bearer ${{ secrets.PAT_TOKEN }}"\
+          -H "Accept: application/vnd.github+json" \
+          -H "Content-Type: application/json" \
+          https://api.github.com/repos/${{ secrets.REPOSITORY_NAME }}/actions/workflows/${{ secrets.WORKFLOW_ID_PRODUCTION }}/dispatches \
+          --data '{"ref":"main", "inputs": { "lakefs_image": ${{ steps.version.outputs.tag }} }}'
```

Comment:
@idanovo this doesn't allow you to create a dependency between the workflows (and test the outcome of the change).

Reply:
@ortz you are right. changed to one one job with dependency